packages/angular/src/abstract-control.ts

```diff
@@ -86,6 +87,15 @@ export abstract class JsonFormsAbstractControl<
     this.triggerValidation();
   }
 
+  shouldShowUnfocusedDescription(): boolean {
+    const config = this.jsonFormsService.getState().jsonforms.config;
+    const appliedUiSchemaOptions = merge({}, config, this.uischema.options);
+    if (appliedUiSchemaOptions) {
+      return !!appliedUiSchemaOptions.showUnfocusedDescription;
+    }
+    return false;
```

Comment:
Here we hide away the default value for `showUnfocusedDescription` in this method. I would prefer when the service would return the default `config` when the user did not set their own. This way we also don't need the `undefined` check and can just immediately return the value of the config.

Reply:
The config here already contains the default values for undefined properties (the default values are set by the configReducer https://github.com/eclipsesource/jsonforms/pull/1584/files#diff-42788974ca6b5be621ff5d539f3155e8R83). I guess I added the undefined checks out of habit, would you prefer if I removed them?

Reply:
Maybe I'm missing something but isn't the config `undefined` by default and will only be set when the user actually calls `updateConfig`? Ideally it should always set to its default state when the user did not call `updateConfig`.

Reply:
You're right, I see your point now. The last comit fixes this.